nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -166,7 +166,7 @@ In order to facilitate the secure setup of NiFi, you can use the `tls-toolkit` c
 
 Wildcard certificates (i.e. two nodes `node1.nifi.apache.org` and `node2.nifi.apache.org` being assigned the same certificate with a CN or SAN entry of +*.nifi.apache.org+) are *not officially supported* and *not recommended*. There are numerous disadvantages to using wildcard certificates, and a cluster working with wildcard certificates has occurred in previous versions out of lucky accidents, not intentional support. Wildcard SAN entries are acceptable *if* each cert maintains an additional unique SAN entry and CN entry. 
 
-Potential issues with wildcard certificates:
+==== Potential issues with wildcard certificates:
```

Comment:
With the TOC Level Set to 3, should change this to:

Potential Issues with Wildcard Certificates

Reply:
I wasn't concerned with this being listed in the TOC, but good catch.